### DISCUSSION TITLE
Store values for using in plain js

### DISCUSSION THREAD

**Discussion by miquik** [pmndrs/jotai#2200] [General] [answered]
Hi everyone, I have an atom where I store some values (say for example app settings loaded from server) to be used in React.
But I want to use the same values in the pure JS parts of my program. 
Obviously I can't access atom values outside of React, so I've implemented something like a 'global object storage'. I created a global object (maybe could be a singleton?) that stores values from server so I can use in JS parts of program; then I wrapped this object in a custom SyncStorage so I can create an atomWithStorage atom that reads/writes the global object.

It works, but I would like to know if this is a good idea, if there are any drawbacks and if there are any better methods

Thanks a lot

**Comment by DavidJFelix** [comment, marked as answer]
You can access the atoms outside of react by asking the store directly for the value of an atom. See:

https://jotai.org/docs/guides/migrating-to-v2-api#store-api

**Reply by miquik** [reply]
It makes sense, thank you very much
